charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml

```diff
@@ -27,21 +27,52 @@ rules:
   - get
   - list
   - watch
-  - update
   - create
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  resourceNames:
+  - gardener-extension-{{ .Values.serviceName }}
+  verbs:
+  - update
   - patch
 - apiGroups:
   - resources.gardener.cloud
   resources:
   - managedresources
   verbs:
-  - "*"
+  - list
+  - watch
+  - create
+- apiGroups:
+  - resources.gardener.cloud
+  resources:
+  - managedresources
+  resourceNames:
+  - extension-shoot-rsyslog-relp-configuration-cleaner
+  verbs:
+  - get
+  - update
+  - patch
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
+  - get
 - apiGroups:
   - ""
   resources:
   - configmaps
+  resourceNames:
+  - shoot-rsyslog-relp-dashboards
```

Comment:
For ConfigMaps, we could restrict the `get` to be by `resourceNames` because cache is disabled for ConfigMaps (see above):

```diff
diff --git a/charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml b/charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml
index 5bf8f83..d8c8ecb 100644
--- a/charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml
+++ b/charts/gardener-extension-shoot-rsyslog-relp/templates/clusterrole.yaml
@@ -62,7 +62,6 @@ rules:
   - configmaps
   verbs:
   - create
-  - get
 - apiGroups:
   - ""
   resources:
@@ -70,6 +69,7 @@ rules:
   resourceNames:
   - shoot-rsyslog-relp-dashboards
   verbs:
+  - get
   - update
   - patch
   - delete
```

Reply:
You are right. I forgot about it.